CHANGELOG.md

```diff
@@ -1,7 +1,28 @@
 # Change Log
+
 All notable changes to this project will be documented in this file.
 See the [change log guidelines](http://keepachangelog.com/) for information on how to structure the file.
 
+## [5.1.0]
+
+### Added
+
+- Simplify the setup process with automated creation of deegree configurations and capabilities for WFS/WMS services.
+- Introduced support for the MS Access JDBC driver, expanding connectivity options for your database needs.
+- Migration to Java 17, updating Gradle Docker plugin to 9.3.1 and baseImage to Java 17, ensuring compatibility and leveraging the latest Java features.
+- Updated dependencies addressing security vulnerabilities, safeguarding your system against potential threats.
+- Added support to custom functions for merge statistics, offering more flexibility and customization options.
+
+### Build Improvements
+
+- Fixed the deployArtifact script to ensure reliable deployment of artifacts to the wetransform Artifactory.
+- Introducing an automated build process that generates builds for Linux, Mac, and Windows platforms with every change to the master branch, streamlining development workflows.
+- Migrated Continuous Integration (CI) to GitHub Actions, enhancing development efficiency and automation capabilities.
+
+### Fixed
+
+- Resolved an error occurring when not collecting statistics in the alignment migrator, ensuring smooth operation without disruptions.
```

Comment:
Maybe a style thing - I would prefer to omit these subordinate clauses that add more text but no information.

Maybe rephrase for a clearer communication what the error was about:
"Fixed a problem that lead to alignment migration failing if collection of statistics was not enabled."